config/default/jenkinsio.yaml

```diff
@@ -46,13 +54,14 @@ ingress:
         - path: /zh/
           serviceName: jenkinsio-zh
           servicePort: 80
+  # jenkins.io && www.jenkins.io certificate are managed at Fastly level
   tls:
     - secretName: jenkinsio-tls
       hosts:
-        - jenkins.io
-        - www.jenkins.io
         - jenkins-ci.org
+        - jenkins.io
```

Comment:
The comment and this line with jenkins.io doesn't match up
was jenkins.io bare supposed to be readded?

Reply:
I updated the comment, only www is now managed by Fastly, jenkins.io was initialy configured on Fastly but we don't need it anymore since we use www.jenkins.io